src/jsonld/organization.tsx

```diff
@@ -14,20 +14,20 @@ export interface OrganizationJsonLdProps extends JsonLdProps {
   legalName?: string;
   sameAs?: string[];
   address?: Address | Address[];
-  contactPoints?: ContactPoint[];
+  contactPoint?: ContactPoint[];
 }
 
 function OrganizationJsonLd({
   type = 'Organization',
   keyOverride,
   address,
-  contactPoints,
+  contactPoint,
```

Comment:
To avoid a breaking change we will need to continue to allow `contactPoints`.

We can do this here:

```suggestion
  contactPoint,
  contactPoints,
```